### DISCUSSION TITLE
MacOS as host set to min 10.13 but works on 10.12.6

### DISCUSSION THREAD

**Discussion by Randrianasulu** [86Box/86Box#4559] [General]
well, just tried to compile 86Box git (from ~June 10) on macos host with macports-installed qt5. It compiled fine, but .app icon was crossed over in Finder, showing that "app was compiled for minimum version of os 10.13" . I edited src/CmakeLists and lowered minimum required OS version to 10.12, and recompiled.

Now 86Box starts and at very minimum accept roms, I can see BIOS screens (no OS tried yet).

Is there  strong reason to keep minimum macos host version at 10.13 ?

**Comment by cold-brewed** [comment]
Yes, 10.13 is the [minimum supported version](https://doc.qt.io/qt-5/supported-platforms.html#macos) for Qt5.
